lib/gltfPipeline.js

```diff
@@ -26,13 +29,24 @@ module.exports = {
     processFileToDisk : processFileToDisk
 };
 
-function processJSON(gltf, options, callback) {
+function writeSources(gltf, outputPath, options, callback) {
+    var embed = options.embed;
+    var embedImage = options.embedImage;
+    var basePath = path.dirname(outputPath);
+    async.each(['buffers', 'images', 'shaders'], function(name) {
+        writeSource(gltf, basePath, name, embed, embedImage, callback);
+    });
+}
+
+function processJSON(gltf, outputPath, options, callback) {
```

Comment:
Really its not a perfect solution either way, but I think its more user friendly to not write the files. Also I probably just didn't think through it enough when I created the issue.